tests/Fixer/LanguageConstruct/FunctionToConstantFixerTest.php

```diff
@@ -126,6 +126,30 @@ public function provideTestCases()
                 '<?php echo phpversion(); echo php_sapi_name(); echo M_PI;',
                 ['functions' => ['pi', 'phpversion']],
             ],
+            'get_class => T_CLASS' => [
```

Comment:
Trait context can be fixed as well without any side effects: https://3v4l.org/i6M96

Checking that the call happening in a method makes sense (we don't do this, but I'll fix this), @SpacePossum is it possible or needs significant efforts?